File: PRAK603-2210817120007-NursyahnaPuteri.c
#include <stdio.h>
int main ()
{
    int b1, b2, input, isimatrik;
    scanf ("%d %d",&b1, &b2);
    int x [b1];
    int y [b2];
    if (b1 == b2) {
        for (input= 0; input < b1; input ++) {
            scanf ("%d", &isimatrik);
            x [input] = isimatrik;
        }for (input = 0; input < b2 ; input ++) {
            scanf ("%d", &isimatrik);
            y [input]= isimatrik;
            printf ("%d\t", x [input] * y [input]);
        }
    } else {
        printf ("Jumlah tidak sama");
    } 
    printf ("\n");
}

File: PRAK603-2210817120007-NursyahnaPuteri.py
b1, b2 = map (int,input ().split())
if (b1==b2) :
    x1 = list (map (int, input (). split ()))
    x2 = list (map (int, input ().split ()))
    z = 0
    for i in range (b1) :
        y1 = []
        y1.append (x1 [z : z + b1])
    for i in range (b2):
        y2 = []
        y2.append (x2 [z : z + b2])
        print (y1 [0][i] * y2 [0][i], end=" ")
    print ()
else :
    print ("jumlah tidak sama")